src/librustc/diagnostics.rs

```diff
@@ -112,6 +112,50 @@ reference when using guards or refactor the entire expression, perhaps by
 putting the condition inside the body of the arm.
 "##,
 
+E0009: r##"
+In a pattern, all values that don't implement the `Copy` trait have to be bound
+the same way. The goal here is to avoid binding simultaneous by-move and by-ref
+at the same time.
```

Comment:
I would still like this sentence changed:

`The goal here is to avoid binding simultaneous by-move and by-ref at the same time.`

The "same time" is redundant given the use of "simultaneous".

I can't see anything else to change :ok_hand: